scripts/poweron/DRAC.py

```diff
@@ -1,46 +1,62 @@
-import subprocess, sys, os.path
+#!/usr/bin/env python3
+
+import os.path
+import subprocess
+import sys
+
 
 class DRAC_NO_SUPP_PACK(Exception):
     """Base Exception class for all transfer plugin errors."""
+
     def __init__(self, *args):
         Exception.__init__(self, *args)
-        
+
+
 class DRAC_POWERON_FAILED(Exception):
     """Base Exception class for all transfer plugin errors."""
+
     def __init__(self, *args):
         Exception.__init__(self, *args)
 
+
 def run2(command):
-   run = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    run = subprocess.Popen(
+        command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

Comment:
Doesn't universal_newlines or encoding need to be set to open the pipes in text mode?

Reply:
Now I am using xcp.cmd instead.